utils/mkdocs_tx.py

```diff
@@ -0,0 +1,178 @@
+#!/usr/bin/env python
+
+import argparse
+import copy
+
+from ruamel.yaml import YAML
+
+# This scripts helps with translatable content from mkdocs.yml
+# It provides commands:
+#   * to create the YAML translatable file
+#   * to update the mkdocs.yml with the translated content
+
+
+def read_config(file_path: str):
+    yaml = YAML(typ="rt")
+    yaml.preserve_quotes = True
+    with open(file_path) as f:
+        return yaml.load(f)
+
+
+def nav_config(config):
+    _nav_config = {}
+
+    def add_nav_entry(title, content):
+        if title:
+            _nav_config[title] = title
+        for _entry in content:
+            if type(_entry) == str:
+                # this is pointing to a page directly, skipping
+                continue
+            for _title, _content in _entry.items():
+                add_nav_entry(_title, _content)
+
+    add_nav_entry(None, config["nav"])
+
+    return _nav_config
+
+
+def get_site_description(config, source_language):
+    site_description = None
```

Comment:
Just curious, I saw in other functions you prefix the local vars with `_`, but here you do not? Am I missing some special meaning of these?